frontend/src/scenes/onboarding/Onboarding.tsx

```diff
@@ -353,9 +354,7 @@ const SessionReplayOnboarding = (): JSX.Element => {
     return (
         <OnboardingWrapper>
             <OnboardingInstallStep
-                usersAction="recording sessions"
                 sdkInstructionMap={SessionReplaySDKInstructions}
-                subtitle="Choose the framework your frontend is built on, or use our all-purpose JavaScript library. If you already have the snippet installed, you can skip this step!"
```

Comment:
These values were never being consumed in the `AlternativeSDKs` or `SDKs` components. Removed them to clean things up and maintain behaviour. Could also just update the components to thread these through. Happier to do whatever growth think is best here